README.md

```diff
@@ -18,6 +18,13 @@ build the source code, run the tests, and contribute your changes to the project
 
 ## How to install
 
+### macOS/Linux installation via Homebrew
+
+Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is now availabe as a brew formula:
```

Comment:
```suggestion
Thanks to [Efrat19](https://github.com/efrat19), arduino-cli is availabe as a brew formula since version `0.5.0`:
```
I would remove the temporal reference, as will be outdated soon and may be confusing, and use a version reference.

Reply:
Also: `available`. Typo.